### PR TITLE
[Leaderboard] Fix lint errors

### PR DESCRIPTION
**Pull request by MartinMinkov**
While trying to deploy the leaderboard cron job locally via minikube, I was getting yaml parse errors. This PR fixes these parse errors to get the deployment going.

### CODE REVIEW

File: helm/leaderboard/templates/leaderboard-cron-job.yaml [hunk 1]
```diff
@@ -1,28 +1,28 @@
 apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
-  name: { { .Values.leaderboard.name } }
+  name: {{ .Values.leaderboard.name }}
 spec:
-  schedule: { { .Values.leaderboard.schedule } }
+  schedule: "* */1 * * *"
```
Comment (lk86) [inline helm/leaderboard/templates/leaderboard-cron-job.yaml:6]: I liked that the schedule was configurable, what did helm lint say about it?

Comment (MartinMinkov) [reply helm/leaderboard/templates/leaderboard-cron-job.yaml:6]: Yeah, I'm not sure why I had to change it like this. I was getting the following errors:

`[ERROR] templates/leaderboard-cron-job.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 6: did not find expected alphabetic or numeric character

[ERROR] templates/leaderboard-cron-job.yaml: object name does not conform to Kubernetes naming requirements: ""`

I figured I'd try sticking the value to see if it'd pass and it did. I'll look into this further.

Comment (lk86) [reply helm/leaderboard/templates/leaderboard-cron-job.yaml:6]: Ahh yeah, I think you want `"{{ .Values.leaderboard.schedule }}"` or `{{ .Values.leaderboard.schedule | quote }}`

**Review by lk86 — APPROVED** [review]
Approved pending the schedule fix

**Pull request merged**